packages/@react-spectrum/tabs/src/Tabs.tsx

```diff
@@ -268,7 +268,6 @@ export function TabList<T>(props: SpectrumTabListProps<T>) {
   let tabListclassName = classNames(styles, 'spectrum-TabsPanel-tabs');
   const tabContent = (
     <div
-      {...styleProps}
```

Comment:
I think we may need to keep this since `styleProps` can return more than just className and style - there's also `hidden`. I guess just only apply it when orientation is vertical?